includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -55,7 +55,7 @@ private function getLangWithFallback( $lang ) {
 			'careers' => 'https://careers.wikia.com',
 			'press' => 'http://fandom.wikia.com/press',
 			'contact' => 'http://fandom.wikia.com/contact',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "contact" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)